simphony/cuds/lattice.py

```diff
@@ -67,6 +69,43 @@ def origin(self):
         return self._origin
 
     def get_node(self, index):
+        pass
```

Comment:
The only logic that the BaseLattice provides is the `__init__` which I think is trivial enough to not require a new parent class. 

I am also not in favor of the BaseLattice overriding the abstract methods. I think is it not the right solution to the *Empty* Lattice problem. I better solution and one that have been discussed in the wiki refers to introducing a `create_lattice` method to the ABCLattice, so one can create lattice instances.